### PR TITLE
Expose `main` via executable.dart

### PR DESCRIPTION
**Pull request by abarth**
Expose the main entry point for the tools via the library lets us run the tools
from the Flutter package, which simplifies the setup for end developers because
they don't need to declare a dependency on sky_tools directly.

### CODE REVIEW

**Comment by abarth** [comment]
@devoncarew

File: bin/sky_tools.dart [hunk 1]
```diff
@@ -2,162 +2,8 @@
...
-    ..addCommand(new InstallCommand())
-    ..addCommand(new ListCommand())
-    ..addCommand(new ListenCommand())
-    ..addCommand(new LogsCommand())
-    ..addCommand(new RunMojoCommand())
-    ..addCommand(new StartCommand())
-    ..addCommand(new StopCommand())
-    ..addCommand(new TraceCommand())
-    ..run(args);
+  executable.main(args);
```
Comment (devoncarew) [inline bin/sky_tools.dart]: `void main(List<String> args) => executable.main(args)` ?

**Comment by devoncarew** [comment]
lgtm!

File: bin/sky_tools.dart (hunk 1 above)
Comment (abarth) [inline bin/sky_tools.dart]: Done.

**Pull request merged**